tensorflow_lite_support/c/task/vision/image_classifier.cc

```diff
@@ -167,8 +174,17 @@ TfLiteClassificationResult* GetClassificationResultCStruct(
 
 TfLiteClassificationResult* TfLiteImageClassifierClassifyWithRoi(
     const TfLiteImageClassifier* classifier,
-    const TfLiteFrameBuffer* frame_buffer, const TfLiteBoundingBox* roi) {
-  if (classifier == nullptr || frame_buffer == nullptr) {
+    const TfLiteFrameBuffer* frame_buffer, const TfLiteBoundingBox* roi,
+    TfLiteSupportError** error) {
+  if (classifier == nullptr) {
+    ::tflite::support::CreateTfLiteSupportError(
+        kInvalidArgumentError, "Expected non null image classifier.", error);
+    return nullptr;
+  }
+
+  if (frame_buffer == nullptr) {
+    ::tflite::support::CreateTfLiteSupportError(
+        kInvalidArgumentError, "Expected non null frame buffer.", error);
     return nullptr;
   }
```

Comment:
We can. But right now `frame_buffer` param of `CreateCppFrameBuffer ` is being passed by reference. So we have to check for `null` before the call. If we can change that to pass the` frame_buffer` as a pointer by value, we can check for `nullptr` in `CreateCppFrameBuffer ` and create a Status with appropriate message and return it.  This will enable us to remove the extra null check in the image_classifer for frame_buffer. Should I change the param to a pointer?